sdk/typescript/src/resources/jobs/pipelines/pipelineJobCreateOrUpdateSample.ts

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import {
+    JobBase,
+  } from "@azure/arm-machinelearning";
+  import { client, getEnvironmentVariable } from "../../../utils";
+  import { createOrUpdateComponentVersion } from "../../component/componentVersionsCreateOrUpdateSample";
+  
```

Comment:
Bad indentations, maybe we can define some eslint rules to standardize formats such as indentation, declaring return types, and so on.

Reply:
Thanks! Fixed this file's indent for now. Will add eslint in future PR.